modules/saml/routing/routes/routes.yml

```diff
@@ -64,6 +64,13 @@ saml-legacy-sp-metadata:
   }
   methods: [GET]
 
+websso-single-sign-on-head:
+  path: /idp/singleSignOnService
+  defaults: {
+    _controller: 'SimpleSAML\Module\saml\Controller\WebBrowserSingleSignOn::headRequestNotAllowed'
+  }
+  methods: [HEAD]
```

Comment:
Instead, use a condition `context.getMethod() not in ["GET", "POST"]`

Reply:
That update is at https://github.com/simplesamlphp/simplesamlphp/pull/2481/commits/dc29a4b7276c7029bf330c73ed23d566e8268df6